src/catchup/ApplyBucketsWork.cpp

```diff
@@ -156,35 +153,6 @@ ApplyBucketsWork::onRun()
         return State::WORK_FAILURE;
     }
 
-    if (mResolveMerges && mDelayTimer)
```

Comment:
consider splitting commits: one of the changes here is to simply remove the old gating code